src/m365/spo/commands/tenant/tenant-settings-set.ts

```diff
@@ -596,6 +604,14 @@ class SpoTenantSettingsSetCommand extends SpoCommand {
       {
         option: '--DisableCustomAppAuthentication [DisableCustomAppAuthentication]',
         autocomplete: ['true', 'false']
+      },
+      {
+        option: '--EnableAzureADB2BIntegration [EnableAzureADB2BIntegration]',
+        autocomplete: ['true', 'false']
+      },
+      {
+        option: '--SyncAadB2BManagementPolicy [SyncAadB2BManagementPolicy]',
+        autocomplete: ['true', 'false']
       }
     ];
```

Comment:
We should include validation to verify that the specified values for both commands are either `true` or `false`

Reply:
The validation already exists, see other comment